lighthouse-core/lib/proto-preprocessor.js

```diff
@@ -16,11 +16,12 @@ const fs = require('fs');
  */
 
 /**
-  * @param {string} result
+  * Transform an LHR into a proto-friendly, mostly-compatible LHR.
+  * @param {LH.Result} lhr
+  * @return {LH.Result}
   */
-function processForProto(result) {
-  /** @type {LH.Result} */
-  const reportJson = JSON.parse(result);
+function processForProto(lhr) {
+  const reportJson = JSON.parse(JSON.stringify(lhr));
```

Comment:
the type error (failure in travis) is because this whole function is operating on an `any` `reportJson` (due to the return type of `JSON.parse`). Luckily there was a `forEach` in there or we wouldn't have noticed the whole thing wasn't being checked :)

Need to keep the `/** @type {LH.Result} */` above this line.